app/models/json_builders/channels_json_builder.rb

```diff
@@ -26,15 +26,32 @@ def initialize
   def build
     channels = []
 
-  [Channel.left_joins(publisher: :site_banner).verified.site_channels, Channel.left_joins(publisher: :site_banner).youtube_channels, Channel.left_joins(publisher: :site_banner).twitch_channels, Channel.left_joins(publisher: :site_banner).twitter_channels].each do |verified_channels|
+  [Channel.verified.site_channels, Channel.youtube_channels, Channel.twitch_channels, Channel.twitter_channels].each do |verified_channels|
       verified_channels.find_each do |verified_channel|
         if @excluded_channel_ids.include?(verified_channel.details.channel_identifier)
           excluded = true
           @excluded_verified_channel_ids.push(verified_channel.details.channel_identifier)
         else
           excluded = false
         end
-        channels.push([verified_channel.details.channel_identifier, true, excluded, verified_channel.publisher&.site_banner&.read_only_react_property || {}])
+
+        # Add banners - if default_site_banner_mode=true use default_site_banner for all channels, else add channel_banner
+        publisher = Publisher.find_by(id: verified_channel.publisher_id)
+
+        if publisher
```

Comment:
I don't think you need to check for this because if a channel doesn't have a publisher then there is something seriously wrong. Too defensive in my opinion